### PR TITLE
Removed http_json_api metrics from ledger metrics [DPP-1307].

### PR DESCRIPTION
**Pull request by atriantafyllos-da**
[DPP-1307](https://digitalasset.atlassian.net/browse/DPP-1307)

### CODE REVIEW

File: ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala [hunk 1]
```diff
@@ -5,14 +5,14 @@ package com.daml.http.dbbackend
 
 import com.daml.lf.data.Ref
 import com.daml.nonempty
-import nonempty.{NonEmpty, +-:}
+import nonempty.{+-:, NonEmpty}
```
Comment (mziolekda) [inline ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala:8]: If possible, I would try not to intervene in the import sequence adopted by other teams. I am guessing the formatter did this automatically?

Comment (atriantafyllos-da) [reply ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala:8]: Yes. I will try to remove it.

**Review by mziolekda — COMMENTED** [review]
Left 2 minor comments. Let me know when addressed. Looks good!

**Comment by atriantafyllos-da** [comment]
> Left 2 minor comments. Let me know when addressed. Looks good!

@mziolekda Addressed them.

File: ledger-service/metrics/src/main/scala/com/digitalasset/http/metrics/HttpJsonApiMetrics.scala [hunk 2]
```diff
@@ -1,20 +1,27 @@
...
+  lazy val ForTesting =
+    new HttpJsonApiMetrics(new MetricRegistry, GlobalOpenTelemetry.getMeter("test"))
+}
+
 class HttpJsonApiMetrics(
-    val prefix: MetricName,
     val registry: MetricRegistry,
     val otelMeter: OtelMeter,
-) {
+) extends DropwizardFactory {
```
Comment (lucb-da) [inline ledger-service/metrics/src/main/scala/com/digitalasset/http/metrics/HttpJsonApiMetrics.scala:23]: What are the reasons for `HttpJsonApiMetrics` to extend `DropwizardFactory` ?

Comment (atriantafyllos-da) [reply ledger-service/metrics/src/main/scala/com/digitalasset/http/metrics/HttpJsonApiMetrics.scala:23]: No reason, I overlooked that the subsequent timers are created by a `DropwizardFactory` defined into the class.

Comment (atriantafyllos-da) [reply ledger-service/metrics/src/main/scala/com/digitalasset/http/metrics/HttpJsonApiMetrics.scala]: Removing it.

**Comment by atriantafyllos-da** [comment]
rebased

**Review by mziolekda — APPROVED** [review]
Thanks, removing those dependencies is a positive move at any measure.

**Review by nicu-da — APPROVED** [review]
LGTM!

**Pull request merged**